<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformation</class>
 <widget class="QWidget" name="Transformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_type_morpho">
     <property name="text">
      <string>Choix du type de transformation :</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_transformation">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>complement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>expansion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>egalisation</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_type_morpho_2">
       <property name="text">
        <string>Valeur minimale :</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slid_mini">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_mini">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_type_morpho_3">
       <property name="text">
        <string>Valeur maximale :</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slid_max">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_max">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Attention ! La fonction transformation ne fonctionne</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>qu'avec une image niveau de gris.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="bout_valider">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slid_mini</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sp_mini</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sp_mini</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slid_mini</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slid_max</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sp_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sp_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slid_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
